fal/models/season.py

```diff
@@ -25,6 +26,7 @@ def __repr__(self) -> str:
         return f"id:{self.id} - {self.season_of_year} {self.year}"
 
     @staticmethod
+    @functools.lru_cache(maxsize=8)
```

Comment:
Do we need to cache 8 seasons? I guess its not a big deal but i don't foresee grabbing information about more than 1 or 2 while running code.